stablehlo/dialect/StablehloOps.td

```diff
@@ -1861,10 +1861,17 @@ def StableHLO_CollectivePermuteOp: StableHLO_Op<"collective_permute",
 
   let arguments = (ins
     HLO_Tensor:$operand,
-    I64ElementsAttr:$source_target_pairs
+    I64ElementsAttr:$source_target_pairs,
+    OptionalAttr<StableHLO_ChannelHandle>:$channel_handle
```

Comment:
No action required, just a notification for future changes that resemble this:

I'm going to spend some time today setting up a backward compatibility testing. Should be relatively simple - bytecode `ops_stablehlo_roundtrip.mlir` at a given time, then future versions of StableHLO will attempt to deserialize that copy. Once we have those tests, changes like this to ops/attrs/types should make a new bytecode copy to `ops_stablehlo_roundtrip_<DATE>.mlir.bc` to ensure backward compatibility.

This change _shouldn't_ have any backward compatibility impact. It will have forward compatibility impact, but we haven't ratified any forward compat requirements yet.